orca/service_test.go

```diff
@@ -151,11 +147,15 @@ func (s) TestE2E_CustomBackendMetrics_OutOfBand(t *testing.T) {
 		default:
 		}
 
+		mu.Lock()
```

Comment:
why did you add this? Please provide inline comments explaining such changes. The previous wantProto should work as is I think. If not, then please mention what is happening